src/Events/Integrations/Plugins/Elementor/Provider.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace TEC\Events\Integrations\Plugins\Elementor;
+
+use TEC\Events\Integrations\Integration_Abstract;
+use TEC\Events\Integrations\Plugins\Plugin_Integration;
+use Tribe__Events__Main;
+
+/**
+ * Class Provider
+ *
+ * @since   TBD
+ *
+ * @package TEC\Events\Integrations\Plugins\Elementor
+ */
+class Provider extends Integration_Abstract {
+	use Plugin_Integration;
+
+	/**
+	 * @inheritDoc
+	 */
+	public static function get_slug(): string {
+		return 'elementor';
+	}
+
+	/**
+	 * @inheritDoc
+	 *
+	 * @return bool Whether or not integrations should load.
+	 */
+	public function load_conditionals(): bool {
+		return defined( 'ELEMENTOR_PATH' ) && ! empty( ELEMENTOR_PATH );
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	protected function load(): void {
+		add_filter( 'elementor/query/query_args', [ $this, 'suppress_query_filters' ], 10, 1 );
+	}
+
+	/**
+	 * Modifies the Elementor posts widget query arguments to set 'tribe_suppress_query_filters' to true for the Event post type.
+	 *
+	 * @param array $query_args The Elementor posts widget query arguments.
+	 *
+	 * @return array The modified Elementor posts widget query arguments.
+	 */
+	public function suppress_query_filters( $query_args ): array {
+		// Bail if the selcted post type is not the Events post type.
+		if ( (array) $query_args['post_type'] !== [ Tribe__Events__Main::POSTTYPE ] ) {
```

Comment:
Post Type could include more than one Post Type, switch this to a `in_array()` check

Reply:
> Post Type could include more than one Post Type, switch this to a `in_array()` check

@bordoni I've created a pr [here](https://github.com/the-events-calendar/the-events-calendar/pull/4223) to address this.